src/App.js

```diff
@@ -1,13 +1,75 @@
 import React from 'react';
-
 import './App.scss';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './components/GoodList';
+
+class App extends React.Component {
+  state = {
+    goods: [],
+  };
+
+  setAllGoods = () => {
+    getAll()
+      .then((goodsFromServer) => {
+        this.setState({
+          goods: goodsFromServer,
+        });
+      });
+  }
+
+  set5Goods = () => {
+    get5First()
+      .then((goodsFromServer) => {
+        this.setState({
+          goods: goodsFromServer,
+        });
+      });
+  }
+
+  setRedGoods = () => {
+    getRedGoods()
+      .then((goodsFromServer) => {
+        this.setState({
+          goods: goodsFromServer,
+        });
+      });
+  }
+
```

Comment:
Could you make 1 function instead of these 3? 
It can accept a callback as a parameter.